data.json

```diff
@@ -14,7 +14,8 @@
   "Facebook": {
     "url": "https://www.facebook.com/{}",
     "urlMain": "https://www.facebook.com/",
-    "errorType": "status_code"
+    "errorType": "status_code",
+    "regexCheck": "^([a-zA-Z0-9.]){5,}$"
```

Comment:
Length of 5 wouldn't be true for facebook, since the `facebook.com/zuck` username is exists.

Reply:
So, what do I need to do?
Should I delete the regex for Facebook?
Modify it as mr. hoadlck suggested _"^[a-zA-Z0-9](?:[a-zA-Z0-9\.]|-(?=[a-zA-Z0-9\.])){4,49}(?<!.com|.org|.net)$"_.
Or modify it in order for it to allow sequencies <50 letters and not contain .com .org .net?